index.js

```diff
@@ -10,6 +10,11 @@ const reactRules = {
   'react/no-children-prop': ERROR,
   'react/display-name': OFF,
   'react/prop-types': OFF,
+  'react/jsx-curly-brace-presence': [
+    ERROR,
+    { props: 'never', children: 'never', propElementValues: 'always' },
+  ],
+  'sort-imports': ['error', { ignoreDeclarationSort: true }],
```

Comment:
Could this conflict with [sku's import order sorting rules](https://github.com/seek-oss/sku/blob/05e598034c5a06db6d0a7436dc2c41db50c8a452/packages/sku/config/eslint/importOrderConfig.js)?

Reply:
I think that's a mis-copy from where we first implemented it. This one can come out @felixhabib